website/docs/r/lb_backend_address_pool_address.html.markdown

```diff
@@ -36,21 +36,42 @@ resource "azurerm_lb_backend_address_pool_address" "example" {
   virtual_network_id      = data.azurerm_virtual_network.example.id
   ip_address              = "10.0.0.1"
 }
+
+data "azurerm_lb_backend_address_pool" "backend-pool-cr" {
+  name            = "globalLBBackendPool"
+  loadbalancer_id = data.azurerm_lb.example.id
+}
+
+resource "azurerm_lb_backend_address_pool_address" "example-1" {
+  name                         = "address1"
+  backend_address_pool_id      = data.azurerm_lb_backend_address_pool.backend-pool-cr.id
+  backend_address_ip_config_id = azurerm_lb.backend-lb-R1.frontend_ip_configuration[0].id
+}
+
+resource "azurerm_lb_backend_address_pool_address" "example-2" {
+  name                         = "address2"
+  backend_address_pool_id      = data.azurerm_lb_backend_address_pool.backend-pool-cr.id
+  backend_address_ip_config_id = azurerm_lb.backend-lb-R2.frontend_ip_configuration[0].id
+}
 ```
 
 ## Arguments Reference
 
--> **Note:** Backend Addresses can only be added to a `Standard` SKU Load Balancer.
+-> **Note:** Backend Addresses can only be added to a `Standard` SKU Load Balancer. Cross region load balancer is for Load Balancer with `Global` SKU.
 
 The following arguments are supported:
 
 * `backend_address_pool_id` - (Required) The ID of the Backend Address Pool. Changing this forces a new Backend Address Pool Address to be created.
 
-* `ip_address` - (Required) The Static IP Address which should be allocated to this Backend Address Pool.
+* `ip_address` - (Optional) The Static IP Address which should be allocated to this Backend Address Pool.
 
 * `name` - (Required) The name which should be used for this Backend Address Pool Address. Changing this forces a new Backend Address Pool Address to be created.
 
-* `virtual_network_id` - (Required) The ID of the Virtual Network within which the Backend Address Pool should exist.
+* `virtual_network_id` - (Optional) The ID of the Virtual Network within which the Backend Address Pool should exist.
+
+* `backend_address_ip_config_id` - (Optional) The ip config ID of the regional load balancer that's added to the global load balancer's backend address pool.
+
+-> **Note:** For cross-region load balancer, please append the name of the load balancers, virtual machines, and other resources in each region with a -R1 and -R2.
```

Comment:
is this a hard requirement? should we be validating this?

Reply:
This is an official guidance mentioned in this [docs](https://docs.microsoft.com/en-us/azure/load-balancer/tutorial-cross-region-portal#prerequisites). From the actual behavior, the regional load balancer without the -R1/-R2 suffix can still be added to the backend address pool. I assume this restriction is taking place during the actual usage, not in resource deployment phase.

Reply:
weird but 🤷‍♀️ - can be adjusted later 👍